src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -486,14 +487,27 @@ function MoneyRequestView({
                                     )
                                 }
                                 brickRoadIndicator={
-                                    getErrorForField('tag', {
-                                        tagListIndex: index,
-                                        tagListName: name,
-                                    })
+                                    getErrorForField(
```

Comment:
`getErrorForField` is called 2 times with the same arguments so we can create a variable to call once.